integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -2181,4 +2210,107 @@ public static String getDeletionTimestamp(String namespace, String pod) throws E
     ExecResult result = ExecCommand.exec(kcmd);
     return result.stdout().trim();
   }
+
+  /**
+   * Check the expected status for provided helm chart.
+   *
+   * @param chartName  helm chart name
+   * @param chartNS helm chart namespace
+   * @param expr expected status
+   * @throws Exception if expected status not found.
+   */
+  public static void checkHelmChartStatus(String chartName, String chartNS, String expr) throws Exception {
+    String cmd = "helm status " + chartName + " --namespace " + chartNS;
+    ExecResult result = ExecCommand.exec(cmd);
+    checkCmdInLoop(cmd, expr, chartName);
+  }
+
+  /**
+   * Retrieve info for provided helm chart.
```

Comment:
you are not returning anything from this method, so "retrieve info" seems incorrect